addons/refuel/functions/fnc_takeNozzle.sqf

```diff
@@ -56,7 +56,14 @@ if (isNull _nozzle) then { // func is called on fuel truck
             private _ropeTarget = _target;
             if (!(_target isKindOf "AllVehicles")) then {
                 private _helper = QGVAR(helper) createVehicle [0,0,0];
-                _helper attachTo [_target, [0,0,0]];
+                hideObjectGlobal _helper;
```

Comment:
Keep in mind this has to run on the server, there is an event in common that can handle the locality:

`[QEGVAR(common,hideObjectGlobal), [_helper, true]] call CBA_fnc_serverEvent;`